backends/arm/arm_partitioner.py

```diff
@@ -47,9 +53,10 @@ def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
 
 @final
 class ArmPartitioner(Partitioner):
-    compile_spec = []
+    compile_spec: List[CompileSpec]
 
-    def __init__(self) -> None:
+    def __init__(self, compile_spec: List[CompileSpec]) -> None:
+        self.compile_spec = compile_spec
         self.delegation_spec = DelegationSpec(ArmBackend.__name__, self.compile_spec)
```

Comment:
Looks like `self.compile_spec` is no longer needed?
```suggestion
        self.delegation_spec = DelegationSpec(ArmBackend.__name__, compile_spec)
```

Reply:
The rest part looks good to me

Reply:
> Looks like `self.compile_spec` is no longer needed?

Done!